test/sensor_simulator/sensor_simulator.h

```diff
@@ -62,25 +73,27 @@ class SensorSimulator
 	void setMagRate(uint32_t rate){ _mag.setRate(rate); }
 	void setBaroRate(uint32_t rate){ _baro.setRate(rate); }
 	void setGpsRate(uint32_t rate){ _gps.setRate(rate); }
+	void setFlowRate(uint32_t rate){ _flow.setRate(rate); }
```

Comment:
Just thinking, do these functions actually add any value above just calling the `_sensor.setRate(rate)` directly?